tests/CMakeLists.txt

```diff
@@ -75,6 +75,11 @@ if(AVIF_ENABLE_GTEST)
     target_include_directories(avifallocationtest PRIVATE ${GTEST_INCLUDE_DIRS})
     add_test(NAME avifallocationtest COMMAND avifallocationtest)
 
+    add_executable(avifalphamulttest gtest/avifalphamulttest.cc)
```

Comment:
Nit: If we only use four characters, I suggest changing "mult" to "prem" because "prem" is the name used in the file format (HEIF?).